Makefile

```diff
@@ -281,4 +282,4 @@ release-rpms: clean build-rpms
 #   make vendor-console
 vendor-console:
 	GIT_REF=$(GIT_REF) CONSOLE_REPO_PATH=$(CONSOLE_REPO_PATH) hack/vendor-console.sh
-.PHONY: vendor-console
```

Comment:
Because my editor auto-magically adds new lines at the end of each file :dancer: